extensions/wikia/Captcha/CaptchaHooks.class.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Captcha;
+
+class Hooks {
+
+	protected static $handler;
+
+	/**
+	 * Get the global Captcha instance
+	 *
+	 * @return Handler
+	 */
+	static function getInstance() {
+		if ( !self::$handler ) {
+			self::$handler = new Handler();
```

Comment:
Is it redundant to call `Captcha\Handler` since we're in the `Captcha` namespace? Just for my own info. Looks like it is (or else this is a bug).